src/main/java/org/keycloak/webbuilder/Links.java

```diff
@@ -87,4 +87,19 @@ public String getLink(String path) {
         return getRoot() + path + (config.isPublish() ? "" : ".html");
     }
 
+    public String getParentLink(Guides.Guide guide) {
```

Comment:
unused, want to keep it?

Reply:
Intellij doesn't pick up when these methods are called from non-java sources it seems. We rely on this method in `guide-entry.ftl` for the breadcrumb links.